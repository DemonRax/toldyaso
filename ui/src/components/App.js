// @flow
import React from 'react';

export const App = (props: any) => {
  return (
    <div>
      Going to tell ya so...
    </div>
  );
};
