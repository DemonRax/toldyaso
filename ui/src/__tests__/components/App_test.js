// @flow
import React from 'react';
import { shallow } from 'enzyme';
import { App } from '../../components/App';

describe('App', () => {
    it('[snapshot] should render', () => {
        const c = shallow(<App />);
        expect(c).toMatchSnapshot();
    });
});
