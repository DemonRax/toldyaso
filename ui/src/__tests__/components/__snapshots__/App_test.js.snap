// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App [snapshot] should render 1`] = `
<div>
  Going to tell ya so...
</div>
`;
